web/tests/timerender.test.js

```diff
@@ -191,76 +191,91 @@ run_test("get_tz_with_UTC_offset", () => {
 
 run_test("render_now_returns_today", () => {
     const today = date_2019;
+    MockDate.set(today.getTime());
```

Comment:
We probably can skip defining a `today` variable that's only used in one place, and just do `MockDate.set(date_2019.getTime());` here.

Reply:
We're using this on L201 again so I though we should keep the variable

Reply:
Well, we could just access `date_2019` directly there, right?

Reply:
Updated!